less/mixins.less

```diff
@@ -1 +1,25 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
+
+.mixin1(){
+
+    display: flex;
```

Comment:
Good looking Mixins. It would be cool to add more flavor to them and make them bigger with more choices